app/decorators/paginating_decorator.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class PaginatingDecorator < Draper::CollectionDecorator
+  # support for will_paginate
+  delegate :current_page, :total_entries, :total_pages, :per_page, :offset
```

Comment:
https://github.com/drapergem/draper/issues/429#issuecomment-12572503

Reply:
Can [this problem](https://github.com/drapergem/draper/issues/429#issuecomment-30467042) also affect us? I see we use `page_entries_info` at https://github.com/3scale/porta/blob/ad6a20c26bd4e53e68b46a4a9edafac6d801f87f/app/views/forums/public/_detail.html.erb#L22

Reply:
Not sure but I can add it, just in case 👍